ibis/backends/base/sql/alchemy/client.py

```diff
@@ -329,3 +329,104 @@ def _sqla_table_to_expr(self, table):
     def version(self):
         vstring = '.'.join(map(str, self.con.dialect.server_version_info))
         return parse_version(vstring)
+
+    def insert(
+        self,
+        to_table_name: str,
+        database: Optional[str] = None,
+        data_obj=None,
+        from_table_name: Optional[str] = None,
+        if_exists: Optional[str] = 'append',
+    ) -> None:
+        """
+        Insert the given data to a table in backend.
+
+        Parameters
+        ----------
+        to_table_name : string
+            name of the table to which data needs to be inserted
+        database : string, optional
+            name of the attached database that the table is located in.
+        data_obj : pd.Dataframe
+            (Later can also be used for sending in table_expr)
+            data_obj is the dataframe containing data which needs to be
+            inserted to to_table_name
+            (Later when table_expr related implementation is done this
+            variable can be used to take in that data)
+        from_table_name : string, optional
+            name of the table from which data needs to be inserted
+        if_exists : string, default 'append'
+            The values available are: {‘fail’, ‘replace’, ‘append’}
```

Comment:
We should explain what this parameter does, something like:

```suggestion
        if_exists : {‘append’, ‘fail’, ‘replace’}
            With append, the rows will be added to the table. With fail, and exception will be raised if the table exists. With replace, if the table exists, it'll be dropped, and recreated before inserting the rows.
```

Reply:
Done!